src/common/reducers/snap-builds.js

```diff
@@ -1,39 +1,58 @@
 import * as ActionTypes from '../actions/snap-builds';
 import { snapBuildFromAPI } from '../helpers/snap-builds';
 
-export function snapBuilds(state = {
+export const snapBuildsInitialStatus = {
```

Comment:
Exporting initial status for individual repository builds, because they will be undefined in the store until any action creates entry for given repo. So now we can use `snapBuildsInitialStatus` as defaults where we need to.